tensorflow/contrib/cmake/external/eigen.cmake

```diff
@@ -7,7 +7,7 @@
 
 include (ExternalProject)
 
-set(eigen_archive_hash "802d984ade26")
+set(eigen_archive_hash "bbffe8ed5f26")
```

Comment:
Yeah, it would be better if the unpacked archive would be renamed to something fixed, then we wouldn't have to touch all those files.